### DISCUSSION TITLE
efmnhrepatriation - Consultations Module

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/EMu-Documentation#977] [Structural Changes]
This is a discussion to collate and discuss changes that we would like to see in the Consultations module (efmnhrepatriation) in EMu. These may be changes to the module structure (e.g. fields & display) or data (values, standards, workflows, etc).

[OPEN STRUCTURAL CHANGE ISSUES](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22structural+changes%22+label%3Aefmnhrepatriation)

**Comment by magpiedin** [comment]
## Receipt Confirmed field

On the "Interaction" tab (AllIntTab, QryIntTab), add a mail "Receipt Confirmed" Yes/No drop-down field to indicate whether a mail receipt for a message was received.

<img width="630" alt="AllIntTab" src="https://github.com/fieldmuseum/EMu-Documentation/assets/8563362/269408e5-7171-4b19-a104-a828bd3d3ff2">

**Comment by mbrownlee2** [comment]
### Associated records for Consultation Module

Could we get the ability to associate consultation records to other consultation records in EMu, similar to how we use the associations tab in the catalog module? 

This will be helpful if a tribal representative visits the museum and their visit is multi-faceted (they meet with collections, repat, and exhibits), or to record follow up email correspondence if a reply is received months after the initial email was sent.

Example consultation records that I would like to associate are IRNs 28235 and 28236. Example collections records that have been associated together with notes (what I want to be able to do in the consultation module) are IRNs 1310624 and 1310613.

**Reply by mbrownlee2** [reply]
This can be done via attaching Event Records.

HOWEVER, this is a single-value field. **Can we update the InfEvent field on the AllInf3Tab to be a table rather than an atomic field so we can add multiple event records to a single consultation record?**

**Comment by mbrownlee2** [comment]
Can we get a way to search Consultation records by party records across all record types?

**Comment by magpiedin** [comment]
In Consultations, update the attachment-field for erights (ResCreditLineRef_tab - 'Credit Line' on Research Visit tab / AllResTab) to align with upcoming [standard Rights tab changes](https://drive.google.com/drive/folders/1s1btLjXIe4JQmaVK6Q0N2fNA4l-6LwOd).  ( e.g. also display erights.SummaryData in separate column if possible)

**Comment by magpiedin** [comment]
Consider/need to have:  reverse-attachment tab in Catalogue for Consultations [Objects field - efmnhrepatriation.ObjObjectsRef_tab]
